contracts/contracts/rem.system/src/delegate_bandwidth.cpp

```diff
@@ -300,6 +300,13 @@ namespace eosiosystem {
           del_bandwidth_table     del_tbl( _self, from.value );
           const auto& del = del_tbl.get( from.value, "user has no resources" );
           check( current_time_point() > del.staked_time + stake_lock_period, "producer cannot undelegate bandwidth during 180 days");
```

Comment:
@AndriiAndrusyshyn please remove _stake lock_ logic from `del_bandwidth_table` we should keep it in `voter_info`